src/app/map/[id]/Tile/item.buttons.static.tsx

```diff
@@ -1,100 +1,138 @@
 
-  if (newExpandedItemsList.length > 0) {
-    newParams.set("expandedItems", newExpandedItemsList.join(","));
+  if (isExpanded) {
+    // Remove from expanded items
+    const filteredIds = allExpandedItemIds.filter(
+      (id) => id !== item.metadata.dbId,
+    );
+    if (filteredIds.length > 0) {
+      newParams.set("expandedItems", filteredIds.join(","));
+    } else {
+      newParams.delete("expandedItems");
+    }
   } else {
-    newParams.delete("expandedItems");
+    // Add to expanded items
+    const newExpandedIds = [...allExpandedItemIds, item.metadata.dbId];
+    newParams.set("expandedItems", newExpandedIds.join(","));
   }
-  newParams.set("focus", itemId);
-  const href = `${pathname}?${newParams.toString()}`;
-  return href;
+
+  return `?${newParams.toString()}`;
+};
+
+const _createNavigationUrl = (
+  item: HexTileData,
+  currentSearchParams: URLSearchParams,
+): string => {
+  const newParams = new URLSearchParams(currentSearchParams);
+
+  // Set the item as the new center focus
+  newParams.set("focus", item.metadata.coordId);
+
+  return `/map/${item.metadata.dbId}?${newParams.toString()}`;
 };
 
 export const TileButtons = ({
   item,
   scale = 1,
-  pathname,
-  currentSearchParamsString,
   allExpandedItemIds,
   hasChildren,
+  isCenter = false,
 }: TileButtonsProps) => {
+  const searchParams = useSearchParams();
+
   const isExpanded = allExpandedItemIds.includes(item.metadata.dbId);
-  const href = getHref(
-    item,
-    pathname,
-    currentSearchParamsString,
-    allExpandedItemIds,
-  );
-  const ariaLabelForButton = isExpanded
+  const ariaLabelForExpandButton = isExpanded
     ? `Collapse ${item.data.name}`
     : `Expand ${item.data.name}`;
 
-  const buttonPositionClasses =
-    scale >= 2
-      ? "absolute top-[15%] left-1/2 -translate-x-1/2 z-20"
-      : "absolute inset-0 z-20 m-auto";
+  const containerPositionClasses = _getButtonPositioning(scale);
+  const shouldRenderExpandButton = hasChildren || isExpanded;
+  const shouldRenderNavigationButton = !isCenter; // Hide navigation button for center tile
 
-  const shouldRenderButton = hasChildren || isExpanded;
+  // Generate URLs
+  const expandUrl = _getExpandUrl(item, searchParams, allExpandedItemIds);
+  const navigationUrl = _createNavigationUrl(item, searchParams);
 
   // Dynamic button and icon sizes based on scale
-  let buttonSizeClasses = "h-7 w-7";
-  let iconSize = 15;
+  let buttonSizeClasses = "h-6 w-6";
+  let iconSize = 12;
 
-  if (scale === 3) {
-    buttonSizeClasses = "h-14 w-14";
-    iconSize = 30;
+  if (scale >= 2) {
+    buttonSizeClasses = "h-8 w-8";
+    iconSize = 16;
   }
 
-  if (scale === 4) {
-    buttonSizeClasses = "h-28 w-28";
-    iconSize = 60;
+  if (scale === 3) {
+    buttonSizeClasses = "h-12 w-12";
+    iconSize = 24;
   }
 
-  if (!shouldRenderButton) {
-    return null;
+  if (scale >= 4) {
+    buttonSizeClasses = "h-20 w-20";
+    iconSize = 40;
   }
```

Comment:
abstract this in a helper and get something like that: `const { iconSize, buttonSizeClasses } = _getButtonsSizing(scale)`